File: Android/RapidCollect/app/src/main/java/com/originaldevelopment/Model/RapidTVModel.java
package com.originaldevelopment.Model;

public class RapidTVModel {

    private String title, desc, videoUrl;
    private int image;

    public RapidTVModel(String title, String desc, String videoUrl, int image) {
        this.title = title;
        this.desc = desc;
        this.videoUrl = videoUrl;
        this.image = image;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getDesc() {
        return desc;
    }

    public void setDesc(String desc) {
        this.desc = desc;
    }

    public String getVideoUrl() {
        return videoUrl;
    }

    public void setVideoUrl(String videoUrl) {
        this.videoUrl = videoUrl;
    }

    public int getImage() {
        return image;
    }

    public void setImage(int image) {
        this.image = image;
    }
}

File: IOS/Rapid Collect/Rapid Collect/Model/RapidTVModel.swift
//
//  RapidTVModel.swift
//  Rapid Collect
//
//  Created by Vijay Rathore on 01/05/20.
//  Copyright © 2020 OriginalDevelopment. All rights reserved.
//

import Foundation
import UIKit


class RapidTVModel {
    
    private var _title = ""
    private var _desc = ""

    private var _videoUrl = ""
    private var _image = ""
    
    init(title : String, desc : String, videoUrl : String, image : String) {
        _title = title
        _desc = desc
        _videoUrl = videoUrl
       _image = image
    }
    
    var title : String{
        return _title
    }
    
    var desc : String {
        return _desc
    }
    
    var videoUrl : String {
        return _videoUrl
    }
    
    var image : String {
        return _image
    }
    
}
